### ISSUE TITLE
The wathc number is broken

### ISSUE THREAD

**Issue by Bashamega** [EddieHubCommunity/HealthCheck#150]
### Description

Hello:)

The eye icon number should be the number of people watching the repo. but it is just showing 0.

### Screenshots

Health check:
![image](https://github.com/user-attachments/assets/a8c4773f-d69b-4590-96c1-4350e25e2d29)

GitHub:
![image](https://github.com/user-attachments/assets/90171508-0bee-4504-8024-5ab16944f5e7)

**Comment by eddiejaoude** [comment]
This is not the watch, maybe a better icon is needed? It is to show how many hidden items there are on the report

<img width="1309" alt="Screenshot 2024-08-12 at 17 02 16" src="https://github.com/user-attachments/assets/242f7438-2b0f-422c-9c6f-bef2b559c340">

**Comment by Bashamega** [comment]
> This is not the watch, maybe a better icon is needed? It is to show how many hidden items there are on the report
> 
> <img alt="Screenshot 2024-08-12 at 17 02 16" width="1309" src="https://private-user-images.githubusercontent.com/624760/357119287-242f7438-2b0f-422c-9c6f-bef2b559c340.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM0Nzg5MjEsIm5iZiI6MTcyMzQ3ODYyMSwicGF0aCI6Ii82MjQ3NjAvMzU3MTE5Mjg3LTI0MmY3NDM4LTJiMGYtNDIyYy05YzZmLWJlZjJiNTU5YzM0MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMlQxNjAzNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YjdiNjVlNmM1MTM1Yjk0NWQyYmM1OTYxYjg5MjJiZDUyNGY0NTM3NDJkOTQzZDYxZWQxOTdlOTE3Zjk4MWI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.q-mCGEWAlNvxKdYuyylUgm-JmWAPCzPBCjlvMy3M20Y">

Yes, I think a better way to display it would be better. But i think the watch number is a nice feature

**Comment by ahmaddonishyar** [comment]
I like to contribute please assign me.

**Comment by eddiejaoude** [comment]
@ScriptShah I have assigned it to you - please change the icon, as we need to keep the information shown

**Comment by ScriptShah** [comment]
will do and make a pr.
